test/acceptance/member_test.exs

```diff
@@ -18,19 +18,4 @@ defmodule Community.Acceptance.MemberTest do
     assert last_member.company_name == "X-men"
     assert last_member.approved == false
   end
-
-  defp click_on(text) do
-    find_element(:link_text, text)
-    |> click
-  end
-
-  defp click_role(name) do
```

Comment:
Oh yeah this makes sense. I was thinking `click_role` was a domain specific thing (like a user's role or something). I think this is fine in the general helpers like you have it :)